e2e/scripts/st_nested_columns_1.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pandas as pd
+
+import streamlit as st
+
+"""
+# Example 1
+Complex widget layout on one side, chart/image/map on the other side. That's the top use case that's always brought up.
+
+---
+"""
+
+# with st.sidebar:
+col1, col2 = st.columns(2, gap="medium")
+# col1.write("## Column 1")
+# col2.write("## Column 2")
```

Comment:
Can you remove this commented code?

Reply:
Thank you, I've just removed the comments 👍